stable/kubecost-reports-exporter/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: kubecost-reports-exporter
 description: Helm chart for exporting kubernetes cost reports to S3
 home: https://www.kubecost.com
 type: application
-version: 1.1.0
+version: 1.1.???
```

Comment:
@nyambati what is the correct docker image tag here?

Reply:
Updated the image version